ErrorResponses.md

```diff
@@ -0,0 +1,42 @@
+# Blue Button 2.0 API Error Responses
+
+## Overview
+
+This document serves as a supplementary to Blue Button 2.0 API Developer Documents, it gives more details on most commonly error responses and recommended error handling on the client side, e.g. guide lines on retrying the requests on certain errors.
+
+## Error Responses and Client Actions
+
+### Authorization Requests
+
+
+| Status Code    | End Point URL   | Error Message              | Action            | Comments                                               |

```

Comment:
I'm trying to decide if this is just too much information. Should we just highlight which type of requests they could retry on? Everything else feels a bit too much for a sample app. Basically it's a lot of text that essentially sums up to: "Don't do anything".

Reply:
though the purpose is to give more bb2 specifics, since HTTP response codes are already defined.

Can reduce the verbose-ness a bit.

Reply:
Want to take an attempt at trimming it down? Really focus on what they would need to know in order to successfully build an application against BB2.

Reply:
those are frequently used error responses among the full set http status codes, and it contains bb2 specific info, so there are added values, or let me know what should be trimmed? don't mind remove the whole mark down - remember it was added based on some feedback-:

Reply:
My only concern is that this table is a lot of text. I feel like most people wont read it all, and then they are missing out on important information. Were we to trim it down to just when they need to do something (ie retry) then they might catch it. If you feel strongly we can leave it though. Just clean up the typing issues I pointed out and we should be able to get this approved 👍

Reply:
for the enforce of typing, for those places that the type of the var or param is obvious, explicit type is given, but some places e.g. the err var of axios try catch, a single type could not cover all situations, e.g. the error thrown from axios end point call could be AxiosError but it could be something else, there are discussion on this from internet..., also ran across some people's code to handle the multiple error types - but it will pull in more code, may be we could log a backlog for "making node js react sample strong typed" as future improvement? thanks for brought this up.